src/js/server/hcapi/applications.js

```diff
@@ -1,37 +1,164 @@
 const { Router } = require('express');
-const { Hacker, HackerApplication, ApplicationResponse } = require('js/server/models');
+const Sequelize = require('sequelize');
+const { Hacker, HackerApplication, ApplicationResponse, TeamMember, Team, db } = require('js/server/models');
 const { createHttpError } = require('./errors');
+const responseLogic = require('js/server/review/response-logic');
+const fs = require('fs');
+const individualScoreQuery = fs.readFileSync('src/js/server/hcapi/queries/indiv_score_unteamed.sql', 'utf8');
 
 const applicationsRouter = new Router();
 
+// Takes an application and the set of all individual scores and
+// the set of teams average scores
+// returns the score for that application as a float
+function calculateScore(application, individualScores, teamScores) {
```

Comment:
Do we want to break this logic out and put it elsewhere? Other parts of the application are going to want to use it (e.g. `src/js/server/review/response-logic.js`)